app/views/users/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1> WELCOME TO WIKI PAGE</h1>
+<%= form_with model :user do |form|%>
```

Comment:
- [ ] The form should use `@user`. This is a reference from the user's controller. ⬇️ 

![user_form_with](https://user-images.githubusercontent.com/32361963/110919071-e5ac3500-832c-11eb-9fd6-693073aae6fb.png)